test/tts/tts.hpp

```diff
@@ -445,6 +445,21 @@ namespace tts
     }
     using type = decltype(filter_impl(Type {}));
   };
+
+  // Check if a type is contained in a types list
+  template<typename T, typename... Ts>
+  constexpr bool
+  is_one_of(types<Ts...> const&) noexcept
+  {
+    bool found[] = {std::is_same_v<T, Ts>...};
+
+    return [&]<std::size_t... I>(std::index_sequence<I...> const&)
+    {
+      return (false || ... || found[I]);
+    }
+    (std::make_index_sequence<sizeof...(Ts)> {});
+  }
```

Comment:
Yhis shouldn't be in tts

Just use https://github.com/jfalcou/eve/blob/11c8d8c8ee2c9b42fa3a25613d6f5bc1efece991/include/eve/detail/meta.hpp#L75

A;so you don't need the nested lambda

Reply:
Thanks, fixed.